### PR TITLE
Added JSONScalar example to docs

### PR DESCRIPTION
**Pull request by Speedy1991**
Added example for a JSON scalar to the docs

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.  -> Only a documentation change
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes. -> Only a documentation change
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).  -> Only a documentation change

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: docs/types/scalars.md [hunk 1]
```diff
@@ -117,3 +117,49 @@ result = schema.execute_sync("{ base64 }")
...
+```
+
+Returns
+```
+{
+  "data": {
+    "hello": {
+      "a": 1
+    },
+    "someNumbers": [1,2, 3]
```
Comment (BryceBeagle) [inline docs/types/scalars.md:162]:
```suggestion
    "someNumbers": [1, 2, 3]
```

**Review by BryceBeagle — CHANGES_REQUESTED** [review]
Just a minor formatting change :+1:

File: docs/types/scalars.md [hunk 2]
```diff
@@ -117,3 +117,49 @@ result = schema.execute_sync("{ base64 }")
...
+)
+
+```
+
+Usage
+```
+@strawberry.type
+class Query:
+    @strawberry.field
+    def data(self, info) -> GenericScalar:
```
Comment (jkimbo) [inline docs/types/scalars.md:143]:
```suggestion
    def data(self, info) -> JSONScalar:
```

File: docs/types/scalars.md [hunk 3]
```diff
@@ -117,3 +117,49 @@ result = schema.execute_sync("{ base64 }")
...
+```
+{
+  "data": {
+    "hello": {
+      "a": 1
+    },
+    "someNumbers": [1,2, 3]
+  }
+}
+```
```
Comment (jkimbo) [inline docs/types/scalars.md:165]: In the docs we have a custom language that presents queries and responses side by side (for example https://strawberry.rocks/docs/guides/errors#graphql-validation-errors). So this code example can become:

```suggestion
```graphql+response
query ExampleDataQuery {
  data
}
---
{
  "data": {
    "hello": {
      "a": 1
    },
    "someNumbers": [1, 2, 3]
  }
}
```
```

**Review by jkimbo — CHANGES_REQUESTED** [review]

**Comment by Speedy1991** [comment]
Added the requested changes. Is the Bot Release.md comment important? I'm not sure if/what I should add to this file?

**Comment by jkimbo** [comment]
@Speedy1991 don't worry about the release file. It's for code changes not doc changes.

**Review by jkimbo — APPROVED** [review]

**Comment by jkimbo** [comment]
Thanks @Speedy1991 ! Signup for a sticker: https://forms.gle/dmnfQUPoY5gZbVT67

**Pull request merged**